pager.go

```diff
@@ -116,6 +116,7 @@ func (p *pager) Fetch(ctx context.Context, dst interface{}) error {
 		Account         []Account         `xml:"account"`
 		Adjustment      []Adjustment      `xml:"adjustment"`
 		AddOn           []AddOn           `xml:"add_on"`
+		AddOnUsage		[]AddOnUsage	  `xml:"usage"`
```

Comment:
`gofmt` needs to be run on all files in the PR

Reply:
I can only say sorry for forgetting about this! - changed